frontend/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,742 @@
+import circleSlice2 from '@iconify/icons-mdi/circle-slice-2';
```

Comment:
There's a Sidebar/Sidebar.tsx now from the a11y branch.

Reply:
I removed this from the commit where the old Sidebar.tsx was added (`frontend: Mark all user-facing strings as translatable`) and rebased.